drivers/SmartThings/wemo/src/parser.lua

```diff
@@ -30,49 +30,41 @@ end
 
 local function handle_binary_state(device, value)
   -- parses wemo insight style ("8|1611850428|58|...") state, works just fine for a single value too
-  log.debug("binary state value=", value)
   local vals = string.gmatch(value, "%d+")
```

Comment:
So is the idea that it's just a bunch of pipe-delimited numbers then?

Reply:
Yeah, my guess is that they are power/energy numbers from the insight plug, but I have yet to make sense of them, and there's no documentation that I have found.